package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "quiq-chat",
-  "version": "1.3.1",
+  "version": "1.4.1",
```

Comment:
This probably should have technically been a major bump since I changed the name of an API, but since we are the only consumers, I opted out of that.